### PR TITLE
YarnClient to get containerInfo

### PR DESCRIPTION
**Pull request by devtagare**
@PramodSSImmaneni could you please review and merge ?

### CODE REVIEW

File: engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java [hunk 1]
```diff
@@ -485,11 +488,12 @@ public final ContainerInfo getAppMasterContainerInfo()
         String nodeHttpAddress = nmHost + ":" + nmHttpPort;
         if (allocatedMemoryMB == 0) {
           String url = ConfigUtils.getSchemePrefix(conf) + nodeHttpAddress + "/ws/v1/node/containers/" + ci.id;
-          WebServicesClient webServicesClient = new WebServicesClient();
-          try {
-            String content = webServicesClient.process(url, String.class, new WebServicesClient.GetWebServicesHandler<String>());
-            JSONObject json = new JSONObject(content);
-            int totalMemoryNeededMB = json.getJSONObject("container").getInt("totalMemoryNeededMB");
+          try (YarnClient rmClient = new YarnClientImpl()) {
```
Comment (vrozov) [inline engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java:491]: Use YarnClient.createYarnClient()

**Review by vrozov — CHANGES_REQUESTED** [review]
Please update PR with JIRA.

File: engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java (hunk 1 above)
Comment (pramodin) [reply engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java:491]: Looks like @devtagare is not working on this. I will make the change as I am familiar with this issue. I will add my commit on top so his pull request will also be addressed when merged.

**Comment by pramodin** [comment]
Made changes in https://github.com/apache/apex-core/pull/538

**Pull request closed without merging**